### ISSUE TITLE
Gmail showing "this version of chrome is not supported"

### ISSUE THREAD

**Issue by thepaandu** [saenzramiro/rambox#673]
### Steps to reproduce

1. open rambox
2. open gmail
3. notice the issue above

### Expected behavior

updated chrome

### Actual behavior

asks to update chrome

### ENVIRONMENT

> Rambox: v0.5.3

> OS: ubuntu

> Arch: x64

**Comment by AddictArts** [comment]
+1
OS: Mac

**Comment by HoracePGreeley** [comment]
+1
OS: Windows 10
Version: 0.5.3
Electron: 1.4.15
Chromium: 53.0.2875.143

**Comment by clivend** [comment]
from google: "Gmail will continue to function on Chrome Browser v53 and below through the end of the 2017. "

**Comment by ekerazha** [comment]
Same issue.

**Comment by arjun-xcode** [comment]
Same issue here

**Comment by rickkuilman** [comment]
I've got the same problem!

```
Version: 0.5.3
Platform: darwin (x64)
Electron: 1.4.15
Chromium: 53.0.2785.143
Node: 6.5.0
```

**Comment by HarlemSquirrel** [comment]
I have this issue as well. 
Version: 0.5.3
Platform: linux (x64)
Electron: 1.4.15
Chromium: 53.0.2785.143
Node: 6.5.0

**Comment by Aktariel** [comment]
Looks like Rambox is still using Electron 1.4; any plans to upgrade to 1.6?

**Comment by LokiiyLuxe** [comment]
I encounter the same problem !!
This version of the Chrome browser is no longer compatible. Please install a compatible browser.
Can not install anything else ... Rambox not secure ??

**Comment by kprkpr** [comment]
Same here, this version of Chrome in no longer compatible.
Version: 0.5.3
Platform: linux (x64) (Fedora 25)
Electron: 1.4.15
Chromium: 53.0.2785.143
Node: 6.5.0

It works, but having this notification there and knowing it is a bit annoying

**Comment by LokiiyLuxe** [comment]
Excuse me but I still do not understand how to fix the problem?
What is the exact handling to be carried out?
Why is Rambox not updating this problem?

**Comment by nmat** [comment]
@LokiiyLuxe from what I understand the fix is in the new electron release that is in git at the moment. However I have not yet been able to verify it. Rambox had a fix in 0.5.7 that was pulled back due to some issue on MacOS from what I understand also.

**Comment by LokiiyLuxe** [comment]
@nmat 
Thank you for the answer.
Do you advise me to take the version on Git?
It's the same thing ?

**Comment by nmat** [comment]
@LokiiyLuxe issue is that git version is going to be considered in development since it is not a real release. You can always try it to see if it helps with the issue in the gmail message. As mentioned before I have not tried it to 100% but it does use the new electron version so give it a go.

**Comment by dbpolito** [comment]
@saenzramiro 

Any plans to tag/release soon?

**Comment by saenzramiro** [comment]
We are working in next release, hopefully in a few weeks we will release.

**Comment by TiEul** [comment]
@saenzramiro Any update on when a new build could be released, seeing that a lot of commits have been pending since January, including fixes for GMail?

**Comment by TriiNoxYs** [comment]
Check the new [0.5.8 version](https://github.com/saenzramiro/rambox/releases/tag/v0.5.8). 
Gmail bug fixed, this issue can now be closed.

**Comment by TiEul** [comment]
Yeah, works fine now. Thanks a lot @saenzramiro!

**Comment by gingerbeardman** [comment]
strangely gmail never stops loading for me in 0.5.8

![screen shot 2017-05-19 at 13 56 03](https://cloud.githubusercontent.com/assets/49612/26248596/eccfb384-3c9a-11e7-8027-f51dd78a5a9f.png)

**Comment by TriiNoxYs** [comment]
I think you should create a new issue for this problem.
You can use the "Report an Issue" directly into Rambox in the Help menu.

**Comment by saenzramiro** [comment]
This is fixed in v0.5.9, also @gingerbeardman bug was fixed.
